src/task/src/repos/gitHubReposInvoker.ts

```diff
@@ -182,8 +188,8 @@ export default class GitHubReposInvoker extends BaseReposInvoker {
     }
 
     const options: OctokitOptions = {
-      auth: process.env.SYSTEM_ACCESSTOKEN,
-      userAgent: 'PRMetrics/v1.4.0',
+      auth: RunnerInvoker.isGitHub ? process.env.GITHUB_TOKEN : process.env.SYSTEM_ACCESSTOKEN,
```

Comment:
Ideally, referring to the environment variable as SYSTEM_ACCESSTOKEN should be replaced throughout.
In ADO, System.AccessToken provides an OAuth token for the PCBS account which is one of the system "god" accounts, providing unrestricted access to the tenant.
However, my understanding is that this tool functions with both the OAuth System.AccessToken or with a scoped Personal Access Token (which is a very different auth mechanism).

Reply:
By the way, the failure in the current build is happening because the old build is deployed to the AzDO instance (since it comes from the VS Marketplace) but the new variable is the only one receiving a valid access token. This won't be a problem after the new build is deployed, which is being validated with the GitHub tests.